src/bp/ui-studio/src/web/views/Config/index.tsx

```diff
@@ -0,0 +1,465 @@
+          { acceptLabel: 'Unmount' }
+        )
+      }
+
+      if (allow) {
+        await axios.post(`admin/bots/${this.state.botId}`, bot, axiosConfig)
+        toastSuccess('Bot configuration updated successfully')
+        this.setState({ error: undefined, isSaving: false })
+
+        if (disableChanged) {
+          window.location.reload()
+        }
+      } else this.setState({ error: undefined, isSaving: false })
+    } catch (err) {
+      this.setState({ error: err, isSaving: false })
+    }
+  }
+
+  handleInputChanged = event => {
+    this.setState({ [event.target.name]: event.target.value })
+  }
+
+  handleDefaultLangChanged = async event => {
+    const lang = event.target.value
+
+    if (!this.state.selectedDefaultLang) {
+      this.setState({ selectedDefaultLang: lang })
+      return
+    }
+
+    if (this.state.selectedDefaultLang !== lang) {
+      const currentName = this.state.languages.find(x => x.value === this.state.selectedDefaultLang).label
+      const newName = this.state.languages.find(x => x.value === lang).label
+      const conf = await confirmDialog(
+        `Are you sure you want to change the language of your bot from ${currentName} to ${newName}? All of your content elements will be copied, make sure you translate them.`,
+        { acceptLabel: 'Change' }
+      )
+
+      if (conf) {
+        this.setState({ selectedDefaultLang: lang })
+      }
+    }
+  }
+
+  handleLanguagesChanged = langs => {
+    this.setState({ selectedLanguages: langs })
+  }
+
+  handleCommunityLanguageChanged = event => {
+    this.setState({ selectedDefaultLang: event.target.value, selectedLanguages: [{ value: event.target.value }] })
+  }
+
+  handleImageFileChanged = async event => {
+    const targetProp = event.target.name
+    if (!event.target.files) {
+      return
+    }
+
+    if (!event.target.files[0].type.includes('image/')) {
+      this.setState({
+        error: `${targetProp} requires an image file`
+      })
+      return
+    }
+
+    const data = new FormData()
+    data.append('file', event.target.files[0])
+
+    if (this.state.error) {
+      this.setState({ error: null })
+    }
+
+    try {
+      const res = await axios.post(`bots/${this.state.botId}/media`, data, {
+        ...axiosConfig,
+        headers: { 'Content-Type': 'multipart/form-data' }
+      })
+      this.setState({ [targetProp]: res.data.url })
+    } catch (err) {
+      this.setState({ error: err })
+    }
+  }
+
+  handleDetailsCollapseClick = () => {
+    this.setState({ isDetailsOpen: !this.state.isDetailsOpen })
+  }
+
+  handlePicturesCollapseClick = () => {
+    this.setState({ isPicturesOpen: !this.state.isPicturesOpen })
+  }
+
+  render() {
+    const statuses = []
+    for (const status of this.state.statuses) {
+      statuses.push(
+        <option key={status.value} value={status.value}>
+          {status.label}
+        </option>
+      )
+    }
```

Comment:
`const statusOptions = this.state.statusOptions.map(el => (<option>....</option>))`